### ISSUE TITLE
Health Check returns success for exception with empty message

### ISSUE THREAD

**Issue by mknapik** [ianheggie/health_check#22]
I rely on AWS SDK exceptions. For some errors `aws-sdk` returns empty exception, e.g. `Aws::S3::Errors::Forbidden` with no message.

```ruby
      begin
        errors = HealthCheck::Utils.process_checks(checks)
      rescue Exception => e
        errors = e.message
      end     
      # errors == ''
      if errors.blank?
        # acts as everything is fine even though the exception was thrown
      ...
```

**Comment by mknapik** [comment]
I suggest either

* having two variables: for status and message
* `errors = e.message.blank? ? e.class.to_s : e.message` in rescue

**Comment by mknapik** [comment]
@ianheggie Is it gem still maintained? I was thinking about contributing but had problems with running tests.

**Comment by ianheggie** [comment]
It is a while since I have put work into this gem, but I still have an active interest. What problems did you have running tests?

**Comment by mknapik** [comment]
I looked at `.travis.yml` to figure out how to start. I run `./test/test_with_railsapp`, I choose 4.2 and got the following message:

```
Checking .../health_check/test/bin/rails is present ...
Skipped rails since it already exists.
If you want to overwrite skipped stubs, use --force.
rails has no executables, but you may want one from a gem it depends on.
  railties has: rails
  bundler has: bundle, bundler
Test aborted (unable to create .../health_check/test/bin/rails)
```
`ruby --version` `2.2.1`
`gem --version` `2.4.8`
`bundle --version` `1.10.6`

**Comment by ianheggie** [comment]
Released in v2.0.0 (used:  errors = e.message.blank? ? e.class.to_s : e.message.to_s)
